### PR TITLE
fixing some bugs that occur due to API changes in some python packages

### PR DESCRIPTION
**Pull request by rcasero**

### CODE REVIEW

**Review by BryceWayne — APPROVED** [review]
Looks good to me

**Review by BryceWayne — APPROVED** [review]
Looks good. Thanks for this.

File: Batch.py [hunk 1]
```diff
@@ -1,29 +1,24 @@
 import torch
-from torchtext import data
```
Comment (na018) [inline Batch.py:2]: Not sure if it is a good idea to work with legacy modules.
Maybe working with the torch DataLoader could be a solution?

Comment (rcasero) [reply Batch.py:2]: Sure @na018 , feel free to edit the code!

**Pull request merged**